app/screens/AppNavigator/screens/Balances/screens/TokensVsUtxoScreen.tsx

```diff
@@ -0,0 +1,61 @@
+import { MaterialIcons } from '@expo/vector-icons'
+import React from 'react'
+import { View } from 'react-native'
+import { Text } from '../../../../../components'
+import { translate } from '../../../../../translations'
+import { tailwind } from '../../../../../tailwind'
+import { getTokenIcon } from '../../../../../components/icons/tokens/_index'
+
+export function TokensVsUtxoScreen (): JSX.Element {
+  return (
+    <View testID='token_vs_utxo_screen'>
+      <View style={tailwind('pt-6 px-4 border-b-2 border-gray-200')}>
+        <Text style={tailwind('text-black pb-6')}>
+          {translate('screens/ConvertScreen', 'DFI exists in two forms – UTXO and token, which are interchangeable depending on the usage. You can easily convert between the two forms with the app.')}
```

Comment:
`translate('screens/TokensVsUtxos', ...)`

should have its own 'section', esp this page has large amount of text required i18n